Incident Wash-Up

```diff
@@ -0,0 +1,19 @@
+Incident Wash-Ups should be used to discuss issues that have affected our customers and how we can help to avoid similar situations from occurring in the future. The Wash-Up should include all relevant parties at their earliest opportunity after the event. 
+
+Please use this format for all Incident Wash-Ups so that we have a consistent set of data on how we handle situations. Please add all notes from Incident Wash-Ups to the Culture repo, within the Wiki, for reference.
```

Comment:
In SB we've created a type of jira so we can easily link any pieces of work that drop out or people that were involved.  It's internal too so no prying eyes :)